.github/workflows/release.yml

```
@@ -0,0 +1,15 @@
+name: Build Notebook Container
+on:
+  release:
+    types: [published]
```

Let's keep this manual -- This way we won't get surprised if we some day make an alpha/non-stable release.

```suggestion
  workflow_dispatch:
```